src/shared/database/amendments.ts

```diff
@@ -39,7 +39,7 @@ const AMENDMENT_VERSION_REGEX =
   /\| \[(?<amendmentName>[a-zA-Z0-9_]+)\][^\n]+\| (?<version>v[0-9]*\.[0-9]*\.[0-9]*|TBD) *\|/u
 
 export const NETWORKS_HOSTS = new Map([
-  ['main', 'ws://s2.ripple.com:51233'],
+  ['main', 'wss://xrplcluster.com'],
```

Comment:
Why is this changed?

Reply:
We often got rate limit while looping and sending `ledger` and `tx` RPC to clio (not sure if it's because of the recent issue) to backtrack amendments. We can either change this to xrpl-cluster completely or use it as a backup.

Reply:
Has this been discussed with the Clio folks?

Reply:
`RippledError(You are placing too much load on the server., {\"error\":\"slowDown\",\"error_code\":10,\"error_message\":\"You are placing too much load on the server.\",\"status\":\"error` This happens once in a while with Clio

Reply:
Just posted my question on their channel

Reply:
I'm surprised that the testnet and devnet haven't thrown similar "slowdown" errors, given that the amendments' status are expected to be similar between the main, dev and test networks.

Reply:
We are doing this because we are only using the entry node (so one node only) per each network to retrieve live amendments on Flag + 1 ledger. The backtracking would be just in case the entry node is down, we will use Clio. We only fetch Flag +1 ledgers in the last 30 minutes so I dont think it send too many requests to Clio